docker/docker-compose.development.yml

```diff
@@ -91,3 +91,14 @@ services:
     command: ['node', './deploy-subgraph.js']
     depends_on:
       - graph-node
+  wedlocks:
+    image: unlockprotocol/wedlocks
+    command: npm run dev
+    environment: 
+      SMTP_HOST: 'smtp_host'
+      SMTP_USERNAME: 'smtp_username'
+      SMTP_PASSWORD: 'smtp_password'
+      BASE64_WEDLOCKS_PRIVATE_KEY: 'pk'
```

Comment:
This may break some functionality in dev -- I think it will impact the query strings in the generated URLs. @julien51 can confirm

Reply:
Interesting, would we be able to generate the relevant data utilized here explicitly for the local development use case?

Reply:
Sorry for the delay here. Not sure it would actually break? Worth a try...

Reply:
I think you can generate a private key like this (see https://github.com/unlock-protocol/unlock/blob/master/wedlocks/src/__tests__/encrypter.test.js#L11)
```
    forge.rsa.generateKeyPair({ bits: 2048, workers: 2 }, function(
      err,
      keypair
    ) {
      privateKey = forge.pki.privateKeyToPem(keypair.privateKey)
      publicKey = forge.pki.publicKeyToPem(keypair.publicKey)
      done()
    })
```